wcivf/apps/elections/migrations/0025_auto_20190412_1528.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def insert_voting_systems(apps, schema_editor):
+    VotingSystem = apps.get_model("elections", "VotingSystem")
+    db_alias = schema_editor.connection.alias
+    VotingSystem.objects.using(db_alias).bulk_create(
+        [
+            VotingSystem(
+                slug="AMS",
+                name="Additional Member System",
+                wikipedia_url="https://en.wikipedia.org/wiki/Additional_Member_System",
+                description="In an election using the Additional Member System, each voter casts two votes: a vote for a candidate standing in their constituency (with or without an affiliated party), and a vote for a party list standing in a wider region made up of multiple constituencies.",
+            ),
+            VotingSystem(
+                slug="FPTP",
+                name="First-past-the-post",
+                wikipedia_url="https://en.m.wikipedia.org/wiki/First-past-the-post_voting",
+                description="A first-past-the-post (abbreviated FPTP, 1stP, 1PTP or FPP) or winner-takes-all election is one that is won by the candidate receiving more votes than any others.",
+            ),
+            VotingSystem(
+                slug="sv",
```

Comment:
Wasn't me :see_no_evil: 
This is odd, but I'm replicating it because we want the slugs in EE and WCIVF to be the same